README.md

```diff
@@ -1,12 +1,21 @@
 # CARDS Deploy Tool
 
 Deploys a CARDS project Docker container as well as its dependency
-containers as a Docker Compose environment
+containers as a Docker Compose environment.
 
 ## Usage
 
 ```bash
+# Generate a docker-compose file for the specified project
+# Run this to find out more about the supported configurations:
+python3 generate_compose_yaml.py --help
+# For a standalone cards4sparc instance using the latest published version, run:
 python3 generate_compose_yaml.py --oak_filesystem --cards_docker_image ghcr.io/data-team-uhn/cards4sparc
+# Start the containers:
 docker-compose build
 docker-compose up -d
+# Stop the containers and cleanup:
+docker compose down
+docker-compose rm -vf
+./cleanup.sh
```

Comment:
Without running `docker volume prune -f`, volumes will not be deleted. This means that if CARDS was started using a local MongoDB instance (eg. `--mongo_singular`) running these steps and then bringing up a new environment would keep any previously entered data.